### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by rhosmo**
Noticed a few items that required Info Dev editing.

### CODE REVIEW

File: elasticbeanstalk/README.md [hunk 1]
```diff
@@ -10,12 +10,12 @@ To deploy the Alert Logic Agent Container for EB, you need your unique registrat
 3. Copy your unique registration key.
 
 ## Alert Logic Agent Container Definition
-To deploy the Agent Container for EB, you must:
-1. Include the example container definition file from this repository and merge it to your own EB container configuration file (`Dockerrun.aws.json`). The Agent Container will run as side-car container along with your application container.
-2. Include the example `.ebextensions` configuration and merge it to your own EB application source bundle.
+**To deploy the Agent Container for Elastic Beanstalk:**
+1. Downaload the example container definition file from this repository and merge it with your own Elastic Beanstalk container configuration file (`Dockerrun.aws.json`). The Agent Container runs as a sidecar container with your application container.
```
Comment (fss18) [inline elasticbeanstalk/README.md]: i notice typo for Downsload

**Review by fss18 — COMMENTED** [review]
see typo on line 13

**Review by rhosmo — COMMENTED** [review]
Fixed embarassing typo.

**Comment by rhosmo** [comment]
Fixed the embarrassing typo.

**Review by fss18 — APPROVED** [review]
looks good!

**Pull request merged**